### DISCUSSION TITLE
Turn on mathquill by default?

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/webwork2#1732] [General]
I feel like mathquill isn't very well advertised in WW.  If people come to a demo they probably see it, but otherwise they could install WW and never know it's available.

I had two thoughts:

1. Enable it by default in defaults.config.  This would make it apparent to anyone who installs the latest version.
2. Add it to the installation instructions as an option.

Option 1 would get it out there pretty efficiently.  Option 2 will likely miss some people who don't use the installation manual, but would at least get some new adopters.

Thoughts?

**Comment by taniwallach** [comment]
I think this is a good idea. Many courses would probably benefit from using MathQuill by default, and opting out is pretty simple for an instructor to do in a specific course.

**Comment by drgrice1** [comment]
I am for MathQuill by default!

**Comment by Alex-Jordan** [comment]
Can we retire "Still not right" if we do this?

**Comment by drgrice1** [comment]
Yeah, that is a pain.  Although MathQuill is not the only thing that causes a problem for that achievement.  Geogebra applet problems do too.    Also even with MathQuill not enabled, I have seen problems cause the issue if they have enough parts/answers to them.
